### DISCUSSION TITLE
App name change

### DISCUSSION THREAD

**Discussion by TrianguloY** [TrianguloY/URLCheck#80] [Polls]
Following the discussion on https://github.com/TrianguloY/UrlChecker/pull/67 it was decided to change the current app name from a translatable description (Url Checker = Checker of urls) to a fixed same-for-all-languages name, like ClearURL and most other apps do.

Below are some of the possible alternatives, feel free to choose one and/or explain your opinion as comment.

**Comment by TrianguloY** [comment]
The final name will be "URLCheck". 
But I'll keep this poll open in case someone want to discuss further decisions.

**Reply by pressRtowin** [reply]
I was wondering something, specifically due to this app having _URL_ in its name. Despite its primary functionally (checking URLs), the extensions that have been added in order to perform that function better have also pushed it into being quite a formidable handler for "Open with" and "Share" actions, to the point where I typically default to this app now, rather than others like OpenLinkWith.

That being said, there is one shortcoming in this use case, which is outside of the scope of this app's main function of _checking URLs_. There is no ability to handle other types of intents for non text/URL items, such as images for example, and redirect them to your preferred app. It's for this reason I still keep [Better Open With](https://play.google.com/store/apps/details?id=com.aboutmycode.betteropenwith&hl=en_US&gl=US) around.

Would implementing features similar to that of Better Open With make sense for this URLCheck? Despite it not falling under the scope of its name, doing so would make it one of the most powerful tools out there in the broader scope of handling all Open/Share intents. You could even go a step further (though maybe as optional modules for power users) and allow for more fine control over intents such as App Manager does as I mentioned [here](https://github.com/TrianguloY/UrlChecker/issues/31#issuecomment-1437305243).

I can make a feature request for this, but I wanted to ask first since it goes beyond this app's primary function.

**Reply by TrianguloY** [reply]
That's a very long term request, and even though I'm not sure if my free time will allow me, it looks like a good idea. I made the app modular to quickly add new features, and so far it's been helping with it. Unfortunately you need to draw the line somewhere, and I drew it at url manager. Doesn't mean allowing more intents isn't possible, just that's it'll be a bit harder.

Probably a new issue is not the right choice, maybe specific requests but not something that general. But as a new discussion maybe. Yes. I think a new discussion with this comment should allow opinions without any formal request. 
Can you create it, please?
